android/PhysicalWeb/app/src/main/res/layout/list_item_blocked.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="horizontal"
+              android:paddingStart="18dp"
+              android:paddingEnd="18dp"
+              android:paddingTop="18dp"
+              android:paddingBottom="18dp"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:background="#FFFFFF">
+
+    <TextView
+            android:id="@+id/text1"
+            android:layout_width="257dp"
```

Comment:
Instead of setting layout_width or layout_weight to specific values, try this:
```
<TextView>
  android:layout_width="wrap_content"
  android:layout_gravity="start"
  ...
</TextView>
<Button>
  android:layout_width="wrap_content"
  android:layout_gravity="end"
  ...
</Button>
```